src/Core/RenderAPI/Resources/EspShader.hh

```diff
@@ -29,6 +29,11 @@ namespace esp
     void set_attachment_formats(std::vector<EspBlockFormat> formats);
     void set_vertex_layouts(std::vector<EspVertexLayout> vertex_layouts);
     void set_worker_layout(std::unique_ptr<EspUniformMetaData> uniforms_meta_data);
+
+    void only_attach(EspCommandBufferId* id) const;
```

Comment:
co robi only attach?

Reply:
`only_attach` nie używa funkcji `set_viewport` oraz `set_scissors`

Reply:
to może niech attach nie używa funkcji set_viewport/scissors a np attach_with_vs używa?

Reply:
może to tak zostać ale w przyszłości bym zmienił żeby ta nazwa była bardziej wymowna

Reply:
oki